djangosaml2/views.py

```diff
@@ -243,7 +243,8 @@ def assertion_consumer_service(request,
     post_authenticated.send_robust(sender=user, session_info=session_info)
 
     # redirect the user to the view where he came from
-    relay_state = request.POST.get('RelayState', '/')
+    default_relay_state = get_custom_setting('ACS_DEFAULT_REDIRECT_URL', '/')
```

Comment:
I'm OK with this, but how about changing the `'/'` default to [`settings.LOGIN_REDIRECT_URL`](https://docs.djangoproject.com/en/1.9/ref/settings/#login-redirect-url)?